grammar/fbp.peg

```diff
@@ -17,6 +17,93 @@
 
   nodes = {};
 
+  var serialize, indexOf = [].indexOf || function(item) { for (var i = 0, l = this.length; i < l; i++) { if (i in this && this[i] === item) return i; } return -1; };
+
+  parser.serialize = function(graph) {
```

Comment:
Alternative is to not use CoffeeScript at all and just maintain the JS inline in the grammar